FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.JobManagement/AzureStorageJobQueueClient.cs

```diff
@@ -447,7 +465,7 @@ public async Task<bool> KeepAliveJobAsync(JobInfo jobInfo, CancellationToken can
                 else
                 {
                     message = $"Failed to keep alive for job {jobInfo.Id}, the job message is not found.";
```

Comment:
LogInformation instead of LogWarning in Line 670

Reply:
Thanks, fixed it.